### PR TITLE
Update Cloud Trail Permissions

### PR DESCRIPTION
**Pull request by harichintala1**

### CODE REVIEW

File: terraform/environments/bootstrap/member-bootstrap/iam.tf [hunk 1]
```diff
@@ -40,6 +40,10 @@ data "aws_iam_policy_document" "member-access" {
       "cloudtrail:AddTags",
       "cloudtrail:CancelQuery",
       "cloudtrail:Create*",
+      "cloudtrail:StartLogging",
+      "cloudtrail:StopLogging",
```
Comment (davidkelliott) [inline terraform/environments/bootstrap/member-bootstrap/iam.tf:44]:
```suggestion
```
Could we remove the StopLogging action please?  I'm wary of giving permission to turn off Cloudtrail logs! Thanks

**Comment by harichintala1** [comment]
@davidkelliott  I have removed stoptrail - however I have addded,

  "cloudtrail:DeleteTrail" - in view of when we remove resource from Terraform, it may fail to delete the Cloud trail we created ?

File: terraform/environments/bootstrap/member-bootstrap/iam.tf (hunk 1 above)
Comment (harichintala1) [reply terraform/environments/bootstrap/member-bootstrap/iam.tf]: removed 

      "cloudtrail:StopLogging",

**Comment by davidkelliott** [comment]
> @davidkelliott I have removed stoptrail - however I have addded,
> 
> "cloudtrail:DeleteTrail" - in view of when we remove resource from Terraform, it may fail to delete the Cloud trail we created ?

@harichintala1 I don't think we can have this either I'm afraid, we need to ensure that there is no way a Cloudtrail can be stopped or deleted for security reasons.  I think it will have to be if you need to remove a cloudtrail once created you will have to ask one of the MP engineers to manually do it.

**Comment by harichintala1** [comment]
> > @davidkelliott I have removed stoptrail - however I have addded,
> > "cloudtrail:DeleteTrail" - in view of when we remove resource from Terraform, it may fail to delete the Cloud trail we created ?
> 
> @harichintala1 I don't think we can have this either I'm afraid, we need to ensure that there is no way a Cloudtrail can be stopped or deleted for security reasons. I think it will have to be if you need to remove a cloudtrail once created you will have to ask one of the MP engineers to manually do it.

Hi - No problem, I have removed that now !

**Comment by harichintala1** [comment]
@davidkelliott could you review this for latest changes please !

**Review by davidkelliott — APPROVED** [review]

**Pull request merged**